expression/builtin_encryption_vec.go

```diff
@@ -139,11 +145,67 @@ func (b *builtinMD5Sig) vecEvalString(input *chunk.Chunk, result *chunk.Column)
 }
 
 func (b *builtinSHA2Sig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinSHA2Sig) vecEvalString(input *chunk.Chunk, result *chunk.Column) error {
```

Comment:
Please add a comment like `builtinSHA2Sig.evalString`

Reply:
Should I add a comment like

Reply:
> Should I add a comment upon L151? like this
> 
> ```
> // vecEvalString evals SHA2(str, hash_length).
> // See https://dev.mysql.com/doc/refman/5.7/en/encryption-functions.html#function_sha2
> ```

Yes, it's better to add a comment.

Reply:
Should I add a comment upon other vectorized `vecEvalxxxx` functions?